Description: Disable dependency loom
 which requires a `cfg(loom)` which we don't support anyway.
Forwarded: not-needed
Last-Update: 2024-10-10
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -119,2 +119,2 @@
-[target."cfg(loom)".dev-dependencies.loom]
-version = "0.7"
+#[target."cfg(loom)".dev-dependencies.loom]
+#version = "0.7"
